alf/utils/reservoir_sampler.py

```diff
@@ -59,13 +59,13 @@ def __init__(self,
             data_spec (nested TensorSpec): spec for the data item
             K (int): the size of the reservoir set
             speed (int): a bigger speed results in a faster decay of old samples.
-                When `speed` is 0, it results in the original reservoir sampling.
+                When `speed` is 1, it results in the original reservoir sampling.
             name (str): name of the sampler
         """
         super(ReservoirSampler, self).__init__(
             data_spec=data_spec, capacity=K, name=name)
         assert isinstance(K, int) and K > 0
-        assert isinstance(speed, int) and speed >= 0
+        assert isinstance(speed, int) and speed > 0
```

Comment:
I think speed doesn't have to be integer as long as it is greater than or equal to one